src/Formula.elm

```diff
@@ -280,8 +439,9 @@ substFormula σ f =
             in
             subst t
 
-        substTs σ bound =
-            mapResult (substT σ bound)
+        substTs : Substitution -> Set String -> List Term -> Result String (List Term)
+        substTs σ bound lst =
```

Comment:
Mne sa viac pacil ten predchadzajuci point free styl ;)

Reply:
Yoyo, nehnevaj sa, ja to tam celkom aj potrebujem. 😀Je mi zrejmejsie co to robi.